public/locales/de/ui.json

```
@@ -794,7 +896,9 @@
   "shop.messages.purchaseSuccess": "{{itemName}} gekauft!",
   "shop.messages.soundproofing": "Weniger Lärm, mehr Frieden. Harmonie +5",
   "shop.messages.traitUnlocked": "Eigenschaft freigeschaltet: {{traitName}} ({{memberId}})",
+  "shop.messages.unknownItem": "Unbekannter Gegenstand",
```

Biome (2.4.13)

[error] The key shop.messages.unknownItem was already declared.

(lint/suspicious/noDuplicateObjectKeys)